natter-api/src/main/java/com/manning/apisecurityinaction/controller/SpaceController.java

```diff
@@ -12,34 +12,40 @@
 public class SpaceController {
 
   private final Database database;
-
   public SpaceController(Database database) {
     this.database = database;
   }
-
   public JSONObject createSpace(Request request, Response response) {
```

Comment:
Create space function code review:

1. Authorization check is missing
2.  No check for null or empty string for both spaceName & Owner that may cause crash
3. No check for input type for both spaceName & Owner
4. Adding inputs to query with no validation may cause sql injection issues.
5. No error handling for missing input
6. No error handling for forbidden status
7. No unit testing for above function
8. Description of function and parameters are not added (comments)
9. Remove empty lines